docs/observability/metrics.md

```diff
@@ -288,6 +291,9 @@ Total amount of time spent in each state by each vcpu (cpu_time excluding hyperv
 ### kubevirt_vmi_vcpu_wait_seconds_total
 Amount of time spent by each vcpu while waiting on I/O. Type: Counter.
 
+### kubevirt_vmi_vnic_info
+Information about VirtualMachineInstance vNIC interfaces. Type: Gauge.
```

Comment:
@avlitman 
Details of VirtualMachineInstance (VMI) vNIC interfaces, such as vNIC name, binding type, network name, and binding plugin in use for each vNIC of a running instance.

Reply:
fixed